Symantec_docs_code.md

```diff
@@ -0,0 +1,112 @@
+
+* Manual doc build: this happened because the time to set up a complete dev environment was at the time so great. I did use all the dev tools, with an otherwise identical-to-dev workflow: Eclipse for editing code comments, Code Collaborator for review, TeamCity to check that my changes to code files weren't breaking the server build, and Perforce for version control. The one thing I did not do was to build the server locally before doing a personal build on TeamCity (this is a build that lets TeamCity pick up your local changes before they're checked into version control).
+
+* Writer use of dev tools and environment for the work: because the doc content was in the code, this only made sense. It's much easier to explore code and figure out what needs to be documented in comments if you've got a good code explorer to work in; it's easier to get reviews done as part of the code review process; TeamCity let me check that my changes wouldn't break the server builds; and version control ... well, the doc content was living in the code, not just with it, so it had to go into version control. All of our doc content, though, was always checked into version control at some point, including all the HTML and PDF deliverables that were generated from our silo'd doc management system. The difference with this project is that the doc source content was actually in the code, and the server picked up the doc build in a separate process.
+
+## Audience considerations
+
+We needed a writer to own these docs because the audience was not quite like the developers who were writing the code. Good projects also recognize that someone who makes a thing isn't necessarily the best person to explain the thing.
+
+As the writer, however, I was unusually involved with potential customers for the web services we were building. I went on one offsite customer visit with the program manager and product manager, and I listened in on long calls with other customers. I learned the third-party management tools that were looking to integrate with our web services, so as to understand better just what our audience of developers and system administrators really needed to do with our product.
+
+But I also had an internal audience, namely the developers who were writing the code. During the second iteration of the project -- an extension of the original set of web services -- we were more tightly organized into an agile scrum team. I attended standups, helped plan sprints, and sometimes wrote comments ahead of the code, based on discussions with the rest of the team and on my familiarity with the basic server functionality that we were exposing with the web services.
```

Comment:
I'm not super familiar with the customer's problem the product solves, other than probably being in the security space? Maybe one more sentence about the problems the customer faces.